pages/api/ddd/uf/v1/[uf].js

```diff
@@ -0,0 +1,71 @@
+import app from '@/app';
+
+import BaseError from '@/errors/BaseError';
+import InternalError from '@/errors/InternalError';
+import NotFoundError from '@/errors/NotFoundError';
+
+import { getDddsData } from '@/services/ddd';
+
+function getNumbersDDD(dddData) {
+  return dddData.reduce((group, ddd) => {
+    if (!group.some((dddGroup) => dddGroup === ddd.ddd)) {
+      group.push(ddd.ddd);
+    }
+    return group;
+  }, []);
```

Comment:
Para evitar ddds duplicados você pode utilizar um `Set` que fica mais simples

```suggestion
  return Array.from(new Set(dddData.map(item => item.ddd)))
```